src/net/sparkzz/modest/io/config/YAMLConfig.java

```diff
@@ -84,9 +84,7 @@ else if (((Number) tempObject).intValue() == 0)
 	public boolean hasValue(String key) {
 		tempObject = data.get(key);
 
-		if (tempObject != null)
-			return true;
-		return false;
+		return (tempObject != null);
```

Comment:
Yeah, I always forget to simplify that exact code on all my projects. One
thing I noticed is that you changed all the tempObjects. You put them
inside the method bodies..

I only know enough about java from what I've read online from forums and my
own experiences. Is it more efficient to have a single object that just
gets overwritten every time it's called or to just create a new method?
Never really looked to far into it, just made assumptions.
On May 18, 2016 12:56 AM, "Kaleb Haslam" <notifications@github.com> wrote:

> In src/net/sparkzz/modest/io/config/YAMLConfig.java
> <https://github.com/BrendonButler/ModestAPI/pull/6#discussion_r63644872>:
>
> > @@ -84,9 +84,7 @@ else if (((Number) tempObject).intValue() == 0)
> >  	public boolean hasValue(String key) {
> >  		tempObject = data.get(key);
> >
> > -		if (tempObject != null)
> > -			return true;
> > -		return false;
> > +		return (tempObject != null);
>
> Much more convenient and easy-to-read when you simplify return statements
> like this.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/BrendonButler/ModestAPI/pull/6/files/12fdedcfc604601ff50678579f6c5b8df7c1ef73#r63644872>
>